server/data_updates/00023_20200529-155751_archive.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : petr
+# Creation: 2020-05-29 15:57
+
+import re
+import logging
+
+from superdesk.commands.data_updates import DataUpdate
+from superdesk import get_resource_service
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class DataUpdate(DataUpdate):
+
+    resource = 'archive'
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        service = get_resource_service(self.resource)
+        desk = mongodb_database['desks'].find_one({'name': re.compile(r'^XFV')})  # image desk
+        if not desk:
+            logger.warning('desk XFV not found')
+        ids = list(mongodb_collection.find({'state': 'in_progress', 'task.desk': None}, {'_id': True}))
```

Comment:
`mongodb_collection` points to `archive`, I think in this case we need a `published` collection.
Why do query `in_progress` items?

Reply:
good catch, that makes no sense, I think I was testing and didn't have any more published items left for it.. :man_facepalming: